docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
```

Comment:
After a time of looking at this file I have changed my opinion about ignoring. We should add docker-compose.yml in gitignore.  
The reason in following - every time when a developer will work with this service he needs to think about credentials security on pushing changes to this repo. If this file in gitignore, he enters credentials and doesn't worry about them anymore